packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -151,22 +151,11 @@ const OptionEditor: React.FunctionComponent<OptionEditorProps> = ({
 
   const onFocus = () => {
     if (hasAutofix && editorRef.current) {
-      if (editorRef.current.editorContents === '') {
-        rafraf(() => {
-          editorRef.current?.applySnippet('\\{${}}');
-        });
-      }
-    }
-  };
-
-  const onPaste = (event: React.ClipboardEvent<HTMLDivElement>) => {
-    if (hasAutofix && editorRef.current) {
-      const editorContents = editorRef.current.editorContents;
-      const snippet = lenientlyFixQuery(editorContents || '');
-      if (snippet) {
-        editorRef.current.applySnippet(snippet);
-        event.preventDefault();
-      }
+      rafraf(() => {
+        if (editorRef.current?.editorContents === '') {
```

Comment:
In addition to my question about whether we should be doing onpaste, I'm wondering if it isn't better to check that the editor is still empty by the time rafraf fires. In case something delayed like a paste occurred.

Reply:
Yeah, if I had to guess that's probably the root cause of the bug (although I can't reproduce it the same way ticket describes)

Reply:
Yeah I can't reproduce it either, but this sounds a reasonable issue.